jp/manual/graphics/low-level-api/pipeline-state.md

```diff
@@ -62,7 +62,7 @@ var mutablePipelineState = new MutablePipelineState();
 
 // 値を設定して再構築
 // Setting values and rebuilding
-mutablePipelineState.State.BlendState = BlendStates.AlphaBlend
+mutablePipelineState.State.BlendState = BlendStates.AlphaBlend;
 mutablePipelineState.Update
```

Comment:
Would you be ok to add here 'Update()' brackets as well? Just because you did that also for EN version. 🙂

Reply:
I was careless. Thank you.

Reply:
Oh, I disagree. I don't think you were careless at all! 🙂 You're improving the docs for the community, and that's greatly appreciated. If you happened to miss an issue that was already there, it's completely understandable. Your contributions are valuable, and we're thankful for your efforts! 💪